src/svConverter/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ file(GLOB libparserSV_SRC
   "${CMAKE_CURRENT_SOURCE_DIR}/*.cpp"
 )
 
-AddParserToBuild("sv2012" generateParsersSV_GENERATED_SRC ${PARSER_GEN_ONLY_LISTENER})
+AddParserToBuild("sv2012" ${generateParsersSV_GENERATED_SRC} ${PARSER_GEN_ONLY_LISTENER})
```

Comment:
AddParserToBuild is a macro and second parameter is name of the variable where sources are stored. https://github.com/Nic30/hdlConvertor/blob/master/src/CMakeLists.txt#L32

Reply:
Agree my fix does not work. 
A look a the makefile decision show that the antlr source code is regenerate because src/vhdlConvertor/vhdlParsers_GENERATED_SRC is missing

```
Reading makefiles...
Updating makefiles....
Updating goal targets....
 File 'src/vhdlConvertor/CMakeFiles/generateParsersvhdl.dir/build' does not exist.
   File 'generateParsersvhdl' does not exist.
     File 'src/vhdlConvertor/CMakeFiles/generateParsersvhdl' does not exist.
       File 'src/vhdlConvertor/vhdlParsers_GENERATED_SRC' does not exist.
      Must remake target 'src/vhdlConvertor/vhdlParsers_GENERATED_SRC'.
make[2]: Entering directory '/home/tom/prog/git/hdlConvertor/_skbuild/linux-x86_64-2.7/cmake-build'
[ 66%] build vhdlParser
/usr/bin/cmake -E make_directory /home/tom/prog/git/hdlConvertor/src/vhdlConvertor/vhdlParser
/usr/lib/jvm/java-10-oracle/bin/java -jar /home/tom/prog/git/hdlConvertor/antlr4/antlr-4.7.2-complete.jar -Werror -Dlanguage=Cpp -listener -package vhdl -o /home/tom/prog/git/hdlConvertor/src/vhdlConvertor/vhdlParser /home/tom/prog/git/hdlConvertor/grammars/vhdl.g4
```
It look like the name of the variable that contain the source is take as the target of the Makefile rules. So the rule should create this directory otherwise all the source code is recompile.